### DISCUSSION TITLE
Be aware this extension is not "Still don't care about cookies" on the chrome webstore

### DISCUSSION THREAD

**Discussion by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#49] [Announcements]
Someone uploaded a non working fork of IDCAC on the chrome webstore with the name "Still don't care about cookies" (Kind of like the name we decided on but fine). That extension is NOT this fork (and doesn't seem to work from reading the source but maybe it does) but it also has a paywall to use.....

If you where searching for this fork on the chrome webstore it's currently under review and will be uploaded soon.
The url would be:
https://chrome.google.com/webstore/detail/i-still-dont-care-about-c/edibdbjcniadpccecjdfdjjppcpchdlm?hl=en-GB&authuser=0 
(It still is on v1.0.0 which contained a bug)

(Oh and for a note I don't have anything against someone creating another fork and publishing it, just wanted to warn people that it isn't this fork mostly because of the paywall)

**Comment by ArjixWasTaken** [comment]
the description is scuffed btw
![image](https://user-images.githubusercontent.com/53124886/192607229-ea67a8b2-ba4d-4c00-8b16-4896a0ba5bb8.png)

**Reply by ArjixWasTaken** [reply]
might be a firefox issue tbh

**Reply by OhMyGuus** [reply]
Yeah it's a issue I noticed it as well sadly I can't edit it until it has been reviewed again.. 🤦‍♂️

**Reply by ArjixWasTaken** [reply]
> Yeah it's a issue I noticed it as well sadly I can't edit it until it has been reviewed again.. 🤦‍♂️

I share that pain, once you send it for review you can't even cancel it...
